buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -1043,6 +1206,22 @@
     "CliNuGetApiKey": {
       "value": null,
       "isSecret": true
+    },
+    "DistroRid_Ubuntu1704": {
+      "value": "ubuntu.17.04-$(PB_TargetArchitecture)"
+    },
+    "DockerCommonRunArgs_Ubuntu1704": {
+      "value": "--name $(PB_DockerContainerName)$(DockerTag_Ubuntu1704) -v \\\"$(PB_SourcesDirectory):$(PB_GitDirectory)\\\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\\\"$(PB_GitDirectory)\\\" $(DockerImageName_Ubuntu1704)"
```

Comment:
Why the triple slashes?  `\\\`?  We don't use them elsewhere.

Reply:
This is again auto-generated by @chcosta 's tool.